tests/test_messages.py

```diff
@@ -234,12 +235,12 @@ def test_append_text(self) -> None:
         trm = ToolRequestMessage(
             content="stub", tool_calls=[ToolCall.from_name("stub_name")]
         )
-        trm_inplace = trm.append_text("text")
+        trm_inplace = cast(ToolRequestMessage, trm.append_text("text"))
         assert trm.content == trm_inplace.content == "stub\ntext"
         # Check append performs an in-place change by default
         assert trm.tool_calls[0] is trm_inplace.tool_calls[0]
 
-        trm_copy = trm.append_text("text", inplace=False)
+        trm_copy = cast(ToolRequestMessage, trm.append_text("text", inplace=False))
```

Comment:
The issue here is `append_text` is incorrectly typed.

Can you do a separate PR that types `Message.append_text` to return `Self`

And then in this PR let's just type ignore this for now